### PR TITLE
interfaces: bluez: fix dbus rules

### PR DESCRIPTION
**Pull request by ssweeny**
This fixes a few issues with the bluez interface, specifically making the dbus policy permanent, fixing the label to match what we see in snappy, and allowing obexd to claim its dbus name.

### CODE REVIEW

**Comment by zyga** [comment]
Looks good

**Comment by ssweeny** [comment]
Looks like the autopkgtest is hitting some issues downloading snaps. I don't think this is something my branch could have caused, but you're the expert @zyga :smile:

**Comment by morphis** [comment]
Tested this with the snap from https://code.launchpad.net/~ssweeny/+snap/bluez/+build/601 in KVM instance having a BT dongle connect. Works fine, see https://paste.ubuntu.com/15931839/ for details.

File: interfaces/builtin/bluez.go [hunk 1]
```diff
@@ -91,7 +96,7 @@ var bluezConnectedPlugAppArmor = []byte(`
 # Allow all access to bluez service
 dbus (receive, send)
     bus=system
-    peer=(label=bluez5_bluez_*),
+    peer=(label=snap.bluez.*),
```
Comment (niemeyer) [inline interfaces/builtin/bluez.go]: This is hard-coding an interface to be used by a specifically named snap, which doesn't seem to make sense for an interface in a platform.

@zyga can you please work with him to ensure we have the proper content in this template?  I thought we had already discussed this last week?

**Comment by niemeyer** [comment]
That doesn't seem to reflect the conversation we had last week about this problem.

File: interfaces/builtin/bluez.go (hunk 1 above)
Comment (morphis) [inline interfaces/builtin/bluez.go]: From what I got the agreement was that we go with this until we have a proper mechanism in place (which @zyga and @jdstrand are working on) to replace the peer label with the correct label from the connected plug.

Comment (ssweeny) [inline interfaces/builtin/bluez.go]: This was my understanding as well. With this fix we can have a working bluez snap in the store today (as opposed to the broken one that's in there now) while we wait on the proper fix.

Comment (jdstrand) [inline interfaces/builtin/bluez.go]: I would be ok with this but please add a comment like this:
\# FIXME: adjust to use 'snap.<name>.<app>' from the connecting slot

**Comment by zyga** [comment]
I'm working on improving interface details now

**Comment by morphis** [comment]
@jdstrand @zyga @niemeyer So what is the decision? Do you want automatic peer label be fixed before we merge this? I would like to get this merged now and fix those bits afterwards so that we get a working snap back in the store.

**Comment by ssweeny** [comment]
@morphis @zyga sent me a [patch](https://github.com/zyga/snappy/commits/bluez-fix-rules) to stop hardcoding the snap name. I think he's waiting on some input from @jdstrand before giving me the go-ahead to pull it in.

**Comment by morphis** [comment]
@ssweeny Thanks. Once we have this here I will do the same for the networkmanager interface

**Comment by mvo5** [comment]
retest this please

**Comment by jdstrand** [comment]
I think we are deadlocked. I gave @zyga a patch that starts the non-hardcoding, but it required snapd changes to implement correctly which @zyga is working on. But then people said they are waiting on me-- can someone say specifically what this is still waiting on?

**Comment by jdstrand** [comment]
I'm told that the PR in question was https://github.com/zyga/snappy/commit/16228ad739da17d0ff975a1d781db4a53dfdbc79 which I didn't see in my email. I commented on that just now.

**Comment by zyga** [comment]
I'm closing this pull request. I've improved on this branch in #1078

**Pull request closed without merging**